.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  plugins: ['prettier'],
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  rules: {
+    'prettier/prettier': 'error',
```

Comment:
This isn't in the prettier recommended rules?

Reply:
Good thought.  Will verify and remove if unnecessary

Reply:
It is included by the recommended rules.  Can remove